rename-project.bash

```diff
@@ -39,6 +39,15 @@ echo "Performing directory renames"
 # move helm stuff to new name
 mv "helm_deploy/hmpps-template-typescript" "helm_deploy/$PROJECT_NAME"
 
+# change cron job to be random time otherwise we hit rate limiting with veracode
+RANDOM_HOUR=$((RANDOM % (9 - 3 + 1) + 3))
```

Comment:
Unless I'm an idiot (quite possibly) isn't `9 - 3 + 1` always going to evaluate to `7`?